code/__DEFINES/layers.dm

```diff
@@ -269,9 +269,16 @@
 /// Layer for tutorial instructions
 #define TUTORIAL_INSTRUCTIONS_LAYER 5
 
-
+///Layer for lobby menu collapse button
+#define LOBBY_COLLAPSE_BUTTON_LAYER 2
+///Layer for lobby menu background image
 #define LOBBY_BACKGROUND_LAYER 3
+///Layer for lobby menu main buttons (Join/Ready, Observe, Charater Prefs)
 #define LOBBY_BUTTON_LAYER 4
+///Layer for lobby menu shutter, which covers up the menu to collapse/expand it
+#define LOBBY_SHUTTER_LAYER 5
+///Layer for lobby menu buttons that are hanging away from and lower than the main panel
+#define LOBBY_BOTTOM_BUTTON_LAYER 6
```

Comment:
why do these all need bespoke layers? shouldn't you just need two?

Reply:
i didn't really prod these much but two is too little i think, shutter must be above the menu but below the bottom buttons so it goes behind them in the animation properly

Reply:
ok so we need like 3 then?

Reply:
i'll check it out